webpack/webpack5-loader-load-themed-styles/src/test/LoadThemedStylesLoader.test.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { LoadThemedStylesLoader } from '../LoadThemedStylesLoader';
+import LoadThemedStylesMock = require('./testData/LoadThemedStylesMock');
+import getCompiler from './testData/getCompiler';
+
+const MATCH_GENERATED_LOADER_STRING_REGEXP: RegExp = /var\sloader\s\=\srequire\(["'](.+?)["']\)/;
```

Comment:
A snapshot test of the loader output will probably suffice.

Reply:
I'm happy to do both so that we are validating correctness with this and then validating any further changes with the snapshots.